test/unit/spec/common/errorDisplay-Spec.js

```diff
@@ -0,0 +1,232 @@
+define([
+    'common/errorDisplay',
+    'common/props',
+], (ErrorDisplay, Props) => {
+    'use strict';
+    const validInput = Props.make({
+        data: {
+            exec: {
+                jobState: {
+                    error: {
+                        name: 'Oh no',
+                        message: 'Disaster',
+                        error: 'calamity\ndisaster\ndespair',
+                    }
+                }
+            }
+        }
+    });
+    let cssBaseClass;
+
+    describe('The error display module', () => {
+        it('loads', () => {
+            expect(ErrorDisplay).not.toBe(null);
+        });
+
+        it('has expected functions', () => {
+            const functions = ['make', 'defaultAdvice', 'cssBaseClass'];
+            functions.forEach((f) => {
+                expect(ErrorDisplay[f]).toBeDefined();
+            });
+        });
+    });
+
+    describe('The UI module attribute defaultAdvice', () => {
+        it('should return a string about contacting support', () => {
+            expect(ErrorDisplay.defaultAdvice).toContain('kbase.us/support');
```

Comment:
Picky, but this should probably be "www.kbase.us/support" - there's currently issues with "kbase.us" not routing to "www.kbase.us" properly.